app/experimenter/nimbus-ui/src/index.tsx

```diff
@@ -5,16 +5,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./components/App";
+import AppErrorBoundary from "./components/AppErrorBoundary";
+import config, { readConfig } from "./lib/config";
+import sentryMetrics from "./services/sentry";
```

Comment:
Ah whoops 3670 introduced a `lib` folder. Unless we have a reason for keeping both I'm happy if it all goes into `services`.